docs/experimental/extendable-param-files.md

```diff
@@ -0,0 +1,90 @@
+# Using the Extendable Bicep Parameters Feature (Experimental!)
+
+## What is it?
+
+Extendable Bicep Parameter Files is a feature that allows you to extend `.bicepparam` files from another `.bicepparam` file in order to reuse parameters across multiple deployments.
+
+When using extendable bicep parameter files, you will have a main `.bicepparam` file that can be consumed by multiple extended `.bicepparam` files.
+
+## Using
+
+`main.bicep` This is your main bicep file, which will define your parameters for deployment.
+
+```bicep
+param namePrefix string
+param location string
+param symbolicName string
+```
+
+`root.bicepparam` This is your main bicepparam file, which can be reused by multiple extended .bicepparam files and in multiple deployments.
+
+```bicep
+using none
+
+param namePrefix = 'share'
+param location = 'westus'
+```
+
+`leaf.bicepparam` This is your extended bicepparam file, which will refer to one main.bicep file and one main .bicepparam file. Any parameter value in this file will override all previous values.
+
+```bicep
+using 'main.bicep'
+
+extends 'root.bicepparam'
+
+param namePrefix = 'extend'
+param symbolicName = 'test'
+```
+
+Resolved Values
+| Param | Value |
+| -- | -- |
+| namePrefix | `'extend'` |
+| location | `'westus'` |
+| symbolicName | `'test'` |
+
+**Note**: As `foo` is defined in both `root.bicepparam` and `leaf.bicepparam` files, any parameter values in the **`leaf.bicepparam`** file will override the values of the parameter in **both** the `main.bicep` and `root.bicepparam` files.
```

Comment:
looks like this is no longer relevant in your new examples, but I think it's fair to show an example like this after you've walked through the core usage scenarios. Do you still want to cover that?